habitat_sim/robots/fetch_robot.py

```diff
@@ -31,7 +33,9 @@ def __init__(self, urdf_path, sim, limit_robo_joints=True, fixed_base=True):
             wheel_mtr_vel_gain=1.3,
             wheel_mtr_max_impulse=10.0,
             base_offset=mn.Vector3(0, 0, 0),
-            ctrl_freq=30,
+            base_link_names=['base_link', 'r_wheel_link', 'l_wheel_link',
```

Comment:
Should these be a list or a set?

Reply:
You're right, this should be a `set`. Now fixed.